src/Models/TasksModel.php

```diff
@@ -1,21 +1,63 @@
 <?php
 
-require_once 'src/Services/DatabaseService.php';
+require_once 'src/Entities/TaskEntity.php';
+require_once 'src/Entities/UserEntity.php';
 
 class TasksModel
 {
-
     private PDO $db;
-
     public function __construct(PDO $db)
     {
         $this->db = $db;
     }
+    public function selectTaskById(int $taskId): TaskEntity
+    {
+        $query=$this->db->prepare('SELECT `id`,`project_id`,`user_id`,
+                                        CONCAT(UPPER(SUBSTRING(name,1,1)), LOWER(SUBSTRING(name,2)))
+                                        AS name,`description`,`estimate`,`deadline`
+                                        FROM `tasks` WHERE `id` = :id;');
+        $query->setFetchMode(PDO::FETCH_CLASS, TaskEntity::class);
+        $query->execute(['id' => $taskId]);
+        return $query->fetch();
+    }
+    public function displayTaskUser(int $id): TaskEntity
```

Comment:
Same here - really this method should return `: TaskEntity|false`

Reply:
You should also take display out of the name - models don't display anything, they just deal with data